--- a/python/private/pypi/whl_installer/wheel_installer.py
+++ b/python/private/pypi/whl_installer/wheel_installer.py
@@ -140,7 +140,7 @@
     pip_args = (
         [sys.executable, "-m", "pip"]
         + (["--isolated"] if args.isolated else [])
         + (["download", "--only-binary=:all:"] if args.download_only else ["wheel"])
         + ["--no-deps"]
-        + deserialized_args["extra_pip_args"]
+        + [arg for arg in deserialized_args["extra_pip_args"] if not arg.startswith("--hash")]
     )
 
     requirement_file = NamedTemporaryFile(mode="wb", delete=False)
