--- a/cc/deps/cc_toolchain_deps.bzl
+++ b/cc/deps/cc_toolchain_deps.bzl
@@ -167,4 +167,10 @@
             strip_prefix = "clang+llvm-18.1.8-x86_64-linux-gnu-ubuntu-18.04",
             remote_file_urls = {
-                "lib/libtinfo.so.5": ["https://storage.googleapis.com/ml-sysroot-testing/libtinfo/libtinfo.so.5"],
-                "lib/libtinfo5-copyright.txt": ["https://storage.googleapis.com/ml-sysroot-testing/libtinfo/copyright.txt"],
+                "lib/libtinfo.so.5": [
+                    "https://storage.googleapis.com/ml-sysroot-testing/libtinfo/libtinfo.so.5",
+                    "http://storage.googleapis.com/ml-sysroot-testing/libtinfo/libtinfo.so.5",
+                ],
+                "lib/libtinfo5-copyright.txt": [
+                    "https://storage.googleapis.com/ml-sysroot-testing/libtinfo/copyright.txt",
+                    "http://storage.googleapis.com/ml-sysroot-testing/libtinfo/copyright.txt",
+                ],
             },
